downloader/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "video-dl",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "node --harmony index.js",
```

Comment:
Is the `--harmony` flag required here?

It's only required if you're using a [flagged item (yellow boxes)](http://node.green/)

Reply:
Probably not then. I thought it would be useful to get the full range of features out of node.